tests/baselines/reference/satisfiesScenarios.errors.txt

```diff
@@ -0,0 +1,150 @@
+tests/cases/compiler/satisfiesScenarios.ts(7,22): error TS1360: Type '{ a: string; b: string; }' does not satisfy the expected type 'A'.
+  Object literal may only specify known properties, and 'b' does not exist in type 'A'.
+tests/cases/compiler/satisfiesScenarios.ts(16,9): error TS1360: Type '{ a: number; b: string; x: number; }' does not satisfy the expected type 'Partial<Record<Keys, unknown>>'.
+  Object literal may only specify known properties, and 'x' does not exist in type 'Partial<Record<Keys, unknown>>'.
+tests/cases/compiler/satisfiesScenarios.ts(23,15): error TS2339: Property 'd' does not exist on type '{ a: number; b: string; x: number; }'.
+tests/cases/compiler/satisfiesScenarios.ts(32,9): error TS1360: Type '{ a: number; b: string; x: number; }' does not satisfy the expected type 'Record<Keys, unknown>'.
+  Object literal may only specify known properties, and 'x' does not exist in type 'Record<Keys, unknown>'.
+tests/cases/compiler/satisfiesScenarios.ts(39,15): error TS2339: Property 'd' does not exist on type '{ a: number; b: string; x: number; }'.
+tests/cases/compiler/satisfiesScenarios.ts(55,19): error TS2339: Property 'z' does not exist on type '{ m: boolean; }'.
+tests/cases/compiler/satisfiesScenarios.ts(65,9): error TS2322: Type 'string' is not assignable to type 'boolean'.
+tests/cases/compiler/satisfiesScenarios.ts(75,30): error TS2322: Type '{ r: number; g: number; d: number; }' is not assignable to type 'Color'.
+  Object literal may only specify known properties, and 'd' does not exist in type 'Color'.
+tests/cases/compiler/satisfiesScenarios.ts(101,15): error TS2339: Property 'y' does not exist on type '{ x: number; }'.
+
+
+==== tests/cases/compiler/satisfiesScenarios.ts (9 errors) ====

```

Comment:
Nit - I would place this test under `conformance`.

Reply:
I've moved to conformance